src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -65,4 +70,84 @@ public static List<String> validateEmploymentDates(CaseData caseData) {
     }
+
+    /**
+     * Adds a group of documents to the ET3 response collection so that it can be displayed on the document tab.
+     * @param caseData data for the current case.
+     * @param documentTypeItemList the documents to be added to the document tab.
+     */
+    public static void addDocuments(CaseData caseData, List<DocumentTypeItem> documentTypeItemList) {
+        if (CollectionUtils.isEmpty(documentTypeItemList)) {
+            return;
+        }
+
+        if (CollectionUtils.isEmpty(caseData.getEt3ResponseDocumentCollection())) {
+            caseData.setEt3ResponseDocumentCollection(new ArrayList<>());
+        }
+
+        documentTypeItemList.stream()
+            .filter(d -> !documentExistsOnCollection(caseData.getEt3ResponseDocumentCollection(),
+                d.getValue().getUploadedDocument().getDocumentBinaryUrl()))
+            .forEach(documentToAdd -> saveFileToCollection(caseData, documentToAdd.getValue()));
+    }
+
+    /**
+     * Adds a document to the ET3 response collection so that it can be displayed on the document tab.
+     * @param caseData data for the current case.
+     * @param documentToAdd the document to be added to the document tab.
+     */
+    public static void addDocument(CaseData caseData, UploadedDocumentType documentToAdd) {
+        if (documentToAdd == null) {
+            return;
+        }
+
+        if (caseData.getEt3ResponseDocumentCollection() == null) {
+            caseData.setEt3ResponseDocumentCollection(new ArrayList<>());
+        } else if (
+            documentExistsOnCollection(caseData.getEt3ResponseDocumentCollection(),
+                documentToAdd.getDocumentBinaryUrl())
+        ) {
+            return;
+        }
+
+        DocumentType documentType = new DocumentType();
+        documentType.setUploadedDocument(documentToAdd);
+
+        saveFileToCollection(caseData, documentType);
+    }
+
+    /**
+     * Saves a document of DocumentType to the ET3Response document collection.
+     * @param caseData data for the current case.
+     * @param documentType the document to save
+     */
+    private static void saveFileToCollection(CaseData caseData, DocumentType documentType) {
+        DocumentTypeItem documentTypeItem = new DocumentTypeItem();
+        documentTypeItem.setId(UUID.randomUUID().toString());
+        documentTypeItem.setValue(documentType);
+
+        caseData.getEt3ResponseDocumentCollection().add(documentTypeItem);
+    }
+
+    /**
+     * Checks if a document collection contains a document based on the URL of the document.
+     * @param collectionToCheck a document collection to check.
+     * @param documentUrlToFind the url of the document to be found.
+     * @return true if found, false if not found or the collection passed is null.
+     */
+    private static boolean documentExistsOnCollection(List<DocumentTypeItem> collectionToCheck,
+                                                      String documentUrlToFind) {
```

Comment:
`toCheck` is a bit wordy. Just "documentUrl" and "collection" are fine without the "ToFind" on the end because its superfluous